test/integration/tests.js

```diff
@@ -65,6 +65,34 @@ test('Gets content types', async (t) => {
   t.ok(response.items, 'items')
 })
 
+test('Gets a content type that has resource links', async (t) => {
+  t.plan(4);
+  const response = await client.getContentType("catalog");
+
+  t.ok(response.sys, 'sys');
+  t.ok(response.name, 'name');
+  t.ok(response.fields, 'fields');
+  t.deepEqual(response.fields, [
+      {
```

Comment:
minor nit: do you think we could add a `type: ResourceLink` field to the ContentType as well? Mainly for clarity in the future. We probably don't need the entry to have the field in it.